.chloggen/alder_expand-docs.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
```

Comment:
Since it is just a documentation improvement, very likely we don't need to add a changelog.

Reply:
Removed. Thanks!